env.local.platform.zsh

```diff
@@ -1,25 +1,19 @@
 #!/bin/echo Run: source
 
-source _env.check-docker.zsh || return 1
+source _env.prerequisites.zsh || return 1
+
+
 
 name=$(basename $0); name=$name:r:s/env.//
 echo $name
 source set-prompt-for-env.zsh $name 
 
 # Build ####
-export JAVA_HOME=$(/usr/libexec/java_home -v11)
+export JAVA_HOME=$(/usr/libexec/java_home -v21)
 export DPM_REPO=$HOME/src/streamsets/domainserver-master
 
 # Auth ####
-export ONE_PASSWORD_ITEM=local.platform
```

Comment:
If I run `source env.local.platform.zsh`  in a shell where I previously had run `source env.dev.my-org.zsh`, I get

```
local.platform
Auth set from 1Password
platform.dev.my-org
---
...
```

so I believe that we should be setting `ONE_PASSWORD_ITEM` explicitly

Reply:
Fixed the duplication

Reply:
Also calling source env.dev.my-org.zsh after source env.local.platform.zsh the `ONE_PASSWORD_ITEM` will be set to` local.platform`